compact-changelog.sh

```diff
@@ -0,0 +1,62 @@
+#!/bin/bash
+# Compile all release notes from the `changes` directory into the changelog
+
+set -euo pipefail
+
+if [[ ! $# -eq 1 ]]; then
+ echo "ERROR: Bad command. Usage example: ${0} v1.2.3"
+ exit 1
+fi
+
+CHANGE_DIR=changes
+CHANGELOG=RELEASE_NOTES.md
+RELEASE_VERSION=$1
+DATE=$(date +%Y-%m-%d)
+
+BAD_FILES=$(find changes -depth 1 -not -name "README.md" -and -not -name "add_*" \
```

Comment:
My Ubuntu 18 version of bash is taking issue with the `-depth` argument, telling me 
```
find: paths must precede expression: `1'
```

I've added an `add_change_note.md` file to the `changes` dir and tried to run it as `./compact-changelog.sh v1.2.3`

Reply:
Changed to use mindepth+maxdepth instead, which should work everywhere